components/hab/src/cli.rs

```diff
@@ -1016,9 +1016,8 @@ fn sub_svc_stop() -> App<'static, 'static> {
     )
 }
 
-#[cfg(any(target_os = "linux", target_os = "macos"))]
 fn sub_svc_load() -> App<'static, 'static> {
-    clap_app!(@subcommand load =>
+    let mut sub = clap_app!(@subcommand load =>
```

Comment:
This is a great simplification! I wonder if for the sake of future extensions it's worth leaving configuration:
```rust
#[cfg(any(unix, windows))]
fn sub_svc_load() -> App<'static, 'static> {
```
It's not strictly necessary now, but in the event we ever do extend to other platforms, it's a nice way to have the compiler point out areas of platform-specific consideration.

Reply:
Given how things appear to be defined in Rust right now, I think this would end up being noise right now. It looks like only the most esoteric of platforms fall outside of `unix` and `windows`, and I don't foresee us going there anytime soon.

Good thought, though 👍